### PR TITLE
Feature/int 1409 fix bugs for drawer link

### PR DESCRIPTION
**Pull request by sabbir0179**

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const parameters = {
 }
 
 const providerFn = ({ theme, children }) => {
-  const muTheme = createMuiTheme(theme)
+  const muTheme = createTheme(theme)
```
Comment (Momen-a9s) [inline .storybook/preview.js:23]: I am not sure why we keep changes that, could @r3p0x verify are we need to add?

**Comment by Momen-a9s** [comment]
the branch not working locally 

Screenshot 2021-08-06 at 10 51 40

File: .storybook/preview.js (hunk 1 above)
Comment (sabbir0179) [reply .storybook/preview.js:23]: I'll keep it the way it is!

**Comment by sabbir0179** [comment]
I've fixed the runtime error. @Momen-a9s please check and approve the CR if it works for you now.

**Review by Momen-a9s — APPROVED** [review]
LGTM, Thanks for changes

**Pull request merged**